src/gesv_mixed.cc

```diff
@@ -204,7 +210,9 @@ int64_t gesv_mixed(
             copy( R, X_lo, opts );
 
             // Solve the system A_lo * X_lo = R_lo.
+            Timer t_rgetrs;
             getrs( A_lo, pivots, X_lo, opts );
+            timers[ "gesv_mixed::rgetrs" ] = t_rgetrs.stop();
```

Comment:
This will only capture the time for the last solve.  In my opinion, it'd be more helpful to measure the total cost of each time we hit this line.
(Personally, I would go further and combine the `bgetrf` and `rgetrs` too.  The first one is only special in that our initial guess is the 0 vector.  But, there may be different opinions on that.)

You'll also want to measure (at least) the cost of the `gemm`s since they'll have a similar cost to the solves.

Reply:
@neil-lindquist I was a bit confused on how we wanted to name the timers (bgetrf and rgetrs) within `gesv_mixed` and where to place some of the timers so I wanted to go ahead and push the file up. Thank you for the input. I agree possibly combining both `bgetrf` and `rgetrs` to be a  single timer could work, we could have a single timer that is around the `else` statement where both are included. If we did that, then the timing would include the cost of the `gemms`.